examples/with-next/pages/static-2.js

```diff
@@ -0,0 +1,54 @@
+import { initUrqlClient } from "next-urql";
```

Comment:
Can remove this page, mainly meant to illustrate/give a testing point for reviewers 😅

Reply:
Yea probably worth removing once you're done 😅